### PR TITLE
Configure release workflows

### PR DESCRIPTION
**Pull request by Ladicle**

### CODE REVIEW

File: .github/workflows/manual-release-pr.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    - name: Bump version and push tag
+      id: tag_version
+      uses: mathieudutour/github-tag-action@v6.2
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        custom_tag: ${{ github.event.inputs.custom_tag }}
+        default_bump: minor
+        dry_run: true
+    - name: Update versions
+      run: ./hack/update-versions.sh ${{ steps.tag_version.outputs.new_version }}
```
Comment (kkaneda) [inline .github/workflows/manual-release-pr.yaml:24]:
```suggestion
      run: ./hack/update-chart-versions.sh ${{ steps.tag_version.outputs.new_version }}
```

**Review by kkaneda — APPROVED** [review]
LG!

Please free free to submit PRs and iterate if needed!

Also just wanted to make sure I understand the versioning 

- We will continue to use 0.x.0 for the image tag and Go modules (at least for now)
- We will use 1.0.0 and later for release versions

Is this correct?

**Comment by Ladicle** [comment]
@kkaneda 
Yes, we will continue to use 0.x.0 for the image tag and Go modules for now. Only for the llmariner repository, we'll use a release branch (+ patch version) to practice the release flow. After merging this PR, I will upgrade this component to 1.0.0.

**Comment by kkaneda** [comment]
Sounds good!

File: .github/workflows/manual-release-pr.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    - name: Bump version and push tag
+      id: tag_version
+      uses: mathieudutour/github-tag-action@v6.2
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        custom_tag: ${{ github.event.inputs.custom_tag }}
+        default_bump: minor
+        dry_run: true
+    - name: Update versions
+      run: ./hack/update-chart-versions.sh ${{ steps.tag_version.outputs.new_version }}
```
Comment (Ladicle) [inline .github/workflows/manual-release-pr.yaml:24]:
```suggestion
      run: ./hack/update-chart-version.sh ${{ steps.tag_version.outputs.new_version }}
```

**Pull request merged**